packages/files/file_path.pony

```diff
@@ -54,70 +54,78 @@ class val FilePath
     """
 
   new val create(
-    base: (FilePath | AmbientAuth),
+    base: (AmbientAuth | FileAuth),
     path': String,
     caps': FileCaps val = recover val FileCaps .> all() end)
-    ?
   =>
     """
-    Create a new path. The caller must either provide the root capability or an
-    existing FilePath.
+    Create a new path.
 
-    If the root capability is provided, path' will be relative to the program's
-    working directory. Otherwise, it will be relative to the existing FilePath,
-    and the existing FilePath must be a prefix of the resulting path.
+    Unless absolute, path' will be relative to the program's working directory.
 
-    The resulting FilePath will have capabilities that are the intersection of
-    the supplied capabilities and the capabilities on the parent.
+    Capabilities are as provided, unlike with `FilePath.from(...)`.
```

Comment:
I'd probably reword this a bit, perhaps by moving it to the first line of the documentation. For example: "Create a new path with the given capabilities", or similar.

Reply:
I did try to clarify via rewording and expanding, but I did not reorder the docstring as it follows the order of arguments. First line is with respect to base, second is about path, and the third is about caps. Once I push the changes up please take a look and let me know it is still reads poorly. I am not against reordering per se, but trying to keep docstrings of create and from consistent.